platform-includes/distributed-tracing/custom-instrumentation/browser/javascript.mdx

```diff
@@ -0,0 +1,98 @@
+
+Distributed tracing will be set up automatically if you've <PlatformLink to="/tracing/">Set Up Tracing</PlatformLink>.
+
+If you have not enabled tracing, or you need to propagate traces in other places than outgoing HTTP requests, you can manually set up your application for distributed tracing to work.
+
+## Enabling Automatic Distributed Tracing
+
+To enable distributed tracing for your frontend, add `browserTracingIntegration` to your `Sentry.init()` options as described in the <PlatformLink to="/tracing/instrumentation/automatic-instrumentation/">Automatic Instrumentation</PlatformLink> docs.
+
+If you want to use distributed tracing but not tracing, set the `tracesSampleRate` option to `0`.
```

Comment:
`If you want to use distributed tracing but not tracing`- this is so confusing. We need to explain this in more detail. Even the SDK option doesn't make much sense, because you've set something to 0 yet it is kind of working in a way because of the integration... it's hurting my head.

Reply:
Rewrote this a bit, is it clearer now?

Reply:
It's clearer, yes, but I still have more questions :D added a comment